tests/cypress/support/commands.js

```diff
@@ -334,7 +334,7 @@ Cypress.Commands.add('createResource', (ratelimitlevel, limitinglevel, httpverb,
         cy.get('#api-rate-limiting-operation-level').click();
     }
     cy.get('#add-operation-selection-dropdown').click();
-    cy.contains('li', httpverb).click();
+    cy.get('ul').contains(httpverb).click();
```

Comment:
Let's check if there's a better way like an id or test id

Reply:
the ul is generated by the mui component.